docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+
+###
+# Get a list of applications
+#
+get:
+  description: Get the list of running applications. Several filters can be applied to filter the result list.
+  is: [ secured ]
```

Comment:
The security schema is a different thing - with that you can define how to access the API (e.g. basic auth). The secured trait is attached to all endpoints, that would need that behavior.